dist/ui.js

```diff
@@ -193,7 +193,7 @@ var UI = function (_EventEmitter) {
     value: function prompt(options, cb) {
       var _this2 = this;
 
-      var prompt = undefined;
+      var prompt = void 0;
```

Comment:
I think `void 0` is an artifact of the build process. 

On doing reviews, makes sure you're looking at `src` and not `dist`, as the latter is irrelevant.